jobs/ci-kubernetes-e2e-kops-aws-canary.env

```diff
@@ -1,20 +1,5 @@
 ### builder
 
-# TODO(krzyzacy) refactor some aws.env later
-
-# Fake provider to trick e2e-runner.sh
-KUBERNETES_PROVIDER=kops-aws
-AWS_CONFIG_FILE=/workspace/.aws/credentials
-# This is needed to be able to create PD from the e2e test
-AWS_SHARED_CREDENTIALS_FILE=/workspace/.aws/credentials
-# TODO(zmerlynn): Eliminate the other uses of this env variable
-KUBE_SSH_USER=admin
-LOG_DUMP_USE_KUBECTL=yes
-LOG_DUMP_SSH_KEY=/workspace/.ssh/kube_aws_rsa
-LOG_DUMP_SSH_USER=admin
-LOG_DUMP_SAVE_LOGS=cloud-init-output
-LOG_DUMP_SAVE_SERVICES=protokube
```

Comment:
If all those envs are stable, I would alternatively recommend create a `platforms/aws.env` and it can be shared among different jobs. (We do it for gce and gke already.)